README.md

```diff
@@ -82,13 +84,15 @@ kg_builder = SimpleKGPipeline(
     embedder=embedder,
     entities=entities,
     relations=relations,
-    on_error="CONTINUE",
+    on_error="IGNORE",
```

Comment:
We could also remove it since it is the default behavior now, up to you.

Reply:
Should the default not be to error out?

Reply:
We thought the user experience would be nicer if they do not have to deal with LLM-related errors on the first run.

Reply:
Not a big deal, can stay as it is.